business/src/main/java/org/mskcc/cbio/portal/service/ApiService.java

```diff
@@ -118,25 +119,10 @@
 	}
 
 	public List<MutationSignature> getSampleMutationSignatures(String genetic_profile_id, List<String> sample_ids, int context_size_on_each_side_of_snp) {
```

Comment:
This is an expansion of the older web api, in which all services were supplied through a single service class. We should consider implementing this as a separate standalone stack of classes (in modules "persistence-*", "service", and "web"). It could become a controller in the weblegacy (api-legacy) location.

The reason for considering this is that we are requiring newly developed API code to have unit tests at all three levels, and also because the business module is about to be broken up, and this functionality will be relocated anyway. The ApiController and ApiService classes combine multiple diverse endpoints and should probably be broken up into smaller related components (by related data types) ... or we can wait for them to migrate into the new REST api arrangement.